tc/core/halide2isl.cc

```diff
@@ -35,13 +35,15 @@ using namespace tc::polyhedral::detail;
 
 SymbolTable makeSymbolTable(const tc2halide::HalideComponents& components) {
   // const Stmt& s) {
-  // Collect and categorize all the Variable symbols
+  // Collect and categorize all the Halide Variable symbols as reduction
+  // or index variables
   class BuildSymbolTable : public IRVisitor {
     using IRVisitor::visit;
     std::set<std::string> included;
     void visit(const Variable* op) {
       if (!included.count(op->name)) {
         if (op->param.defined()) {
+          // why is this empty? - what's the point of this?
```

Comment:
I didn't quite understand why this is empty and added a comment. if one of the reviewers can explain it, that'd be great and I can remove this comment as well